Dockerfile

```diff
@@ -586,6 +586,10 @@ RUN wget --quiet https://github.com/pmd/pmd/releases/download/pmd_releases%2F${P
     chmod a+x ktlint && \
     mv "ktlint" /usr/bin/ \
 
+# detekt installation
+    && curl --retry 5 --retry-delay 5 -sSLO https://github.com/detekt/detekt/releases/download/v1.23.3/detekt-cli-1.23.3.zip && \
+    unzip detekt-cli-1.23.3.zip && \ mv detekt-cli-1.23.3/bin /usr/bin && \ chmod a+x /usr/bin/detekt-cli \
```

Comment:
@enciyo it won't work until you use one line by "&& \" :) (like in my previous commit)